charts/icm-web/templates/ingress.yaml

```diff
@@ -15,24 +15,19 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
```

Comment:
I wonder where the ingress class name is set

Reply:
With `.Values.ingress.annotations` like:

```
annotations:
    kubernetes.io/ingress.class: "nginx"
```

This approach is deprecated but is still supported. I added #151 to later use ingressClassName.